### PR TITLE
update 14.0.6

### PR DESCRIPTION
**Pull request by katietz**

### CODE REVIEW

**Comment by ELundby45** [comment]
@katietz Is this ready for review? Or do we need to wait for windows?

**Comment by katietz** [comment]
yes, we can review it

**Review by varlackc — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -120,14 +125,22 @@ outputs:
       summary: clang compiler components in one package for bootstrapping clang
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:125]: Seems like the LLVM license should be `Apache-2.0 WITH LLVM-exception`, see:
- https://releases.llvm.org/14.0.0/LICENSE.TXT
- https://spdx.org/licenses/LLVM-exception.html

But is it compliant with `conda-build` and `anaconda-linter`?

**Review by skupr-anaconda — APPROVED** [review]
LGTM, but it's pretty complicated

**Pull request merged**